gmm/handler.go

```diff
@@ -1558,6 +1558,12 @@ func AuthenticationProcedure(ue *context.AmfUe, accessType models.AccessType) (b
 	ue.AuthenticationCtx = response
 	ue.ABBA = []uint8{0x00, 0x00} // set ABBA value as described at TS 33.501 Annex A.7.1
 
+	if ue.NgKsi.Tsc == models.ScType_NATIVE && ue.NgKsi.Ksi != 7 {
+		// As per the Specification 24.501 - 5.4.1.3.2 Authentication initiation by the network
```

Comment:
BTW, should not 33.501 the correct document to reference?
```suggestion
		// As per the Specification 33.501 - 6.2.3.2 Key identification
```